.github/CODEOWNERS

```diff
@@ -702,7 +702,7 @@ packages/analytics/shippers/elastic_v3/common @elastic/kibana-core
 packages/analytics/shippers/elastic_v3/server @elastic/kibana-core
 packages/analytics/shippers/fullstory @elastic/kibana-core
```

Comment:
This should not be part of the PR...

Reply:
I blame CI
https://github.com/elastic/kibana/pull/144705/commits/39bef9a7828ace46c8e62d02e8576348dc99931c

Reply:
ah, I see 👍🏾

Reply:
Yeah, sorry about that, we had an issue with the validation of codeowners in CI and that allowed a bunch of changes to the file which were invalidated in PRs. Merged main to confirm that these changes were reverted by the merge.